web_api/open_weather/test_open_weather_api.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+from unittest.mock import patch, Mock
+import open_weather_map
+import test_data.mock_data as mock_data
+from collections import OrderedDict
+from pprint import pprint
+
+class TestPark():
+            def __init__(self):
+                self.lat = 44.59824417
+                self.lon = -110.5471695
+                self.park_code = "yell"
+
+class TestAPIProcessing(unittest.TestCase):
+
+    def test_extract_data_valid_json(self):
```

Comment:
This test is failing, it seems somehow that the main or only difference between the two is the times. Is this a failure of the function or is it an issue with the example data given?

Reply:
please pull and check again.